content/en/real_user_monitoring/browser/collecting_browser_errors.md

```diff
@@ -0,0 +1,175 @@
+|-----------------|--------|-------------------------------------------------------------------|
+| `error.source`  | string | Where the error originates from (for example, `console` or `network`).     |
+| `error.type`    | string | The error type (or error code in some cases).                   |
+| `error.message` | string | A concise, human-readable, one-line message explaining the event. |
+| `error.stack`   | string | The stack trace or complementary information about the error.     |
+
+### Network errors
+
+Network errors include information about failing HTTP requests. The following facets are also collected:
+
+| Attribute                      | Type   | Description                                                                             |
+|--------------------------------|--------|-----------------------------------------------------------------------------------------|
+| `error.resource.status_code`             | number | The response status code.                                                               |
+| `error.resource.method`                | string | The HTTP method (for example `POST`, `GET`).           |
+| `error.resource.url`                     | string | The resource URL.                                                                       |
+| `error.resource.url_host`        | string | The host part of the URL.                                                          |
+| `error.resource.url_path`        | string | The path part of the URL.                                                          |
+| `error.resource.url_query` | object | The query string parts of the URL decomposed as query params key/value attributes. |
+| `error.resource.url_scheme`      | string | The protocol name of the URL (HTTP or HTTPS).                                            |
+| `error.resource.provider.name`      | string | The resource provider name. Default is `unknown`.                                            |
+| `error.resource.provider.domain`      | string | The resource provider domain.                                            |
+| `error.resource.provider.type`      | string | The resource provider type (for example `first-party`, `cdn`, `ad`, `analytics`).                                            |
+
+### Source errors
+
+Source errors include code-level information about the error. More information about the different error types can be found in [the MDN documentation][5].
+
+| Attribute       | Type   | Description                                                       |
+|-----------------|--------|-------------------------------------------------------------------|
+| `error.type`    | string | The error type (or error code in some cases).                   |
+
+## Collect errors manually
+
+Monitor handled exceptions, handled promise rejections and other errors not tracked automatically by the RUM SDK with the `addError()` API:
+
+```javascript
```

Comment:
https://github.com/DataDog/dd-docs-tools/wiki/Markdown#code-blocks - ref the new code block as it has the c+p option.